examples/animation/double_pendulum.py

```diff
@@ -71,22 +73,30 @@ def derivs(state, t):
 y2 = -L2*cos(y[:, 2]) + y1
 
 fig = plt.figure(figsize=(5, 4))
-ax = fig.add_subplot(autoscale_on=False, xlim=(-2, 2), ylim=(-2, 1))
+ax = fig.add_subplot(autoscale_on=False,
+                     xlim=(-(L1+L2), (L1+L2)),
+                     ylim=(-(L1+L2), 1.))
```

Comment:
IMHO this is more confusing than helpful. Either leave the constants (it's not critical that the limits are exactly at the range of the pendulum) or calculate

```
L = L1 + L2  # maximal length of the combined pendulum
```
and use
```suggestion
ax = fig.add_subplot(autoscale_on=False, xlim=(-L, L), ylim=(-L, 1))
```

Reply:
+1 - I like using L. I have changed the code accordingly.